go.mod

```diff
@@ -1,15 +1,18 @@
 module github.com/peak-ai/ais-service-discovery-go
 
-go 1.12
+go 1.15
 
 require (
-	github.com/aws/aws-sdk-go v1.20.17
-	github.com/aws/aws-xray-sdk-go v0.9.4
+	github.com/aws/aws-sdk-go v1.34.20
+	github.com/aws/aws-xray-sdk-go v1.1.0
 	github.com/cihub/seelog v0.0.0-20170130134532-f561c5e57575 // indirect
-	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
-	github.com/pkg/errors v0.8.1 // indirect
-	github.com/sirupsen/logrus v1.4.2
-	github.com/stretchr/objx v0.2.0 // indirect
-	github.com/stretchr/testify v1.3.0
-	golang.org/x/sys v0.0.0-20190710143415-6ec70d6a5542 // indirect
+	github.com/peak-ai/local-plane v0.0.0-20200824173714-0b2cc23e8992
```

Comment:
I think this can be removed. Iirc go doesn't automatically manage this file very well, often have to remove unused dependencies.